app/lib/email_checker.rb

```diff
@@ -0,0 +1,649 @@
+class EmailChecker
+  KNOWN_DOMAINS = [
```

Comment:
tu pourrais rajouter un commentaire qui explique 
- comment c'est trié 
- quelle est la volumétrie minimale pour apparaitre (de la dernière)

En fait le mode d'élaboration de cette liste et sur quelle période. Ça nous permettra d'itérer dessus.